### PR TITLE
add Minimize windows tip 

### PR DESCRIPTION
**Pull request by a-amr**
issue #625

### CODE REVIEW

File: pages/Configuring/Uncommon-tips-&-tricks.md [hunk 1]
```diff
@@ -82,6 +82,17 @@ input {
     kb_options = caps:swapescape
 }
 ```
+##  Minimize Windows Using Special Sorkspaces
+you can alway create a special workspace and put whatever you want in it.
```
Comment (vaxerski) [inline pages/Configuring/Uncommon-tips-&-tricks.md:86]: always

File: pages/Configuring/Uncommon-tips-&-tricks.md [hunk 2]
```diff
@@ -82,6 +82,17 @@ input {
...
+However, this is a more intuitive way by using single keybind to minimize
```
Comment (vaxerski) [inline pages/Configuring/Uncommon-tips-&-tricks.md:87]: a single keybind

File: pages/Configuring/Uncommon-tips-&-tricks.md [hunk 3]
```diff
@@ -82,6 +82,17 @@ input {
...
+it or unminimize, every keybind can only minimize one window at a time.
```
Comment (vaxerski) [inline pages/Configuring/Uncommon-tips-&-tricks.md:88]: , but every keybind...

Comment (a-amr) [reply pages/Configuring/Uncommon-tips-&-tricks.md]: thank you 
fixed now

File: pages/Configuring/Uncommon-tips-&-tricks.md (hunk 2 above)
Comment (a-amr) [reply pages/Configuring/Uncommon-tips-&-tricks.md]: done

File: pages/Configuring/Uncommon-tips-&-tricks.md (hunk 1 above)
Comment (a-amr) [reply pages/Configuring/Uncommon-tips-&-tricks.md]: done

File: pages/Configuring/Uncommon-tips-&-tricks.md [hunk 4]
```diff
@@ -82,6 +82,17 @@ input {
     kb_options = caps:swapescape
 }
 ```
+##  Minimize Windows Using Special Sorkspaces
```
Comment (fufexan) [inline pages/Configuring/Uncommon-tips-&-tricks.md:85]:
```suggestion
##  Minimize windows using special workspaces
```

File: pages/Configuring/Uncommon-tips-&-tricks.md [hunk 5]
```diff
@@ -82,6 +82,17 @@ input {
...
+you can always create a special workspace and put whatever you want in it.
```
Comment (fufexan) [inline pages/Configuring/Uncommon-tips-&-tricks.md:86]: Unnecessary.

File: pages/Configuring/Uncommon-tips-&-tricks.md [hunk 6]
```diff
@@ -82,6 +82,17 @@ input {
...
+However, this is a more intuitive way by using a single keybind to minimize
+it or unminimize, but every keybind can only minimize one window at a time.
```
Comment (fufexan) [inline pages/Configuring/Uncommon-tips-&-tricks.md:88]:
```suggestion
This approach uses special workspaces to mimic the "minimize window" function, by using a single keybind to toggle the minimized state.
Note that one keybind can only handle one window.
```

**Review by fufexan — CHANGES_REQUESTED** [review]

File: pages/Configuring/Uncommon-tips-&-tricks.md (hunk 5 above)
Comment (a-amr) [reply pages/Configuring/Uncommon-tips-&-tricks.md]: maybe

File: pages/Configuring/Uncommon-tips-&-tricks.md (hunk 6 above)
Comment (a-amr) [reply pages/Configuring/Uncommon-tips-&-tricks.md]: check it out

**Comment by a-amr** [comment]
also don't forget to check the keybinding it self

**Review by fufexan — APPROVED** [review]

**Pull request merged**